spec/factories/notes.rb

```diff
@@ -0,0 +1,8 @@
+FactoryGirl.define do
+  factory :note do
+    notable { create(:person) }
```

Comment:
Ordinarily, you'll be creating this factory the other way around, you're more likely to want to create traits like `with_notes` on the person factory than to want to create a note that's already attached to something like this.